jsaddle/src/Language/Javascript/JSaddle/Types.hs

```diff
@@ -532,6 +556,18 @@ instance ToJSON SyncCommand where
 instance FromJSON SyncCommand where
   parseJSON = A.genericParseJSON $ aesonOptions "SyncCommand"
 
+data SyncBlockReq
+  = SyncBlockReq_Req TryReq
+  | SyncBlockReq_Result ValId
+  | SyncBlockReq_Throw Int Text
```

Comment:
Could you add a doc for Int here. It should be the sync depth the throw was sent at right?

Reply:
Done